index.html

```diff
@@ -767,19 +859,175 @@ <h2><dfn>CaptureController</dfn></h2>
+                    If {{CaptureController/[[Source]]}} has been
+                    <a data-cite="GETUSERMEDIA#source-stopped">stopped</a>,
+                    [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[DisplaySurfaceType]]}} is neither
+                    {{DisplayCaptureSurfaceType/"browser"}} nor {{DisplayCaptureSurfaceType/"window"}},
+                    [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If {{CaptureController/[[FocusDecisionFinalized]]}} is <code>true</code>,
+                    [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    If too much time has elapsed since the [=capture-session=] started,
+                    the user agent SHOULD set {{CaptureController/[[FocusDecisionFinalized]]}}
+                    to <code>true</code>. The timespan is left up to the user agent,
+                    but it is recommended that a value of one second be used.
+                  </p>
+                </li>
+                <li>
+                  <p>
+                    Run the [=finalize focus decision algorithm=].
+                  </p>
+                </li>
+              </ol>
             </dd>
+            <p>
+              The <dfn>finalize focus decision algorithm</dfn> consists of running
+              the following steps:
+            </p>
+            <ol>
+              <li>
+                <p>
+                  If {{CaptureController/[[FocusDecisionFinalized]]}} is <code>true</code>,
+                  abort these steps.
+                </p>
+              </li>
+              <li>
+                <p>
+                  Set {{CaptureController/[[FocusDecisionFinalized]]}} to <code>true</code>.
+                </p>
+              </li>
+              <li>
+                <p>
+                  If {{CaptureController/[[InitialFocusLost]]}} is <code>true</code>,
+                  abort these steps.
+                </p>
+              </li>
+              <li>
+                <p>
+                  If {{CaptureController/[[DisplaySurfaceType]]}} is neither
+                  {{DisplayCaptureSurfaceType/"browser"}} nor {{DisplayCaptureSurfaceType/"window"}},
+                  abort these steps.
+                </p>
+              </li>
+              <li>
+                <p>
+                  If {{CaptureController/[[FocusBehavior]]}} is NOT
+                  {{CaptureStartFocusBehavior/"focus-captured-surface"}},
+                  abort these steps.
+                </p>
+              </li>
+              <li>
+                <p>
+                  Run the following steps in parallel:
```

Comment:
Is "Run the following steps" just using the same language as elsewhere, or is there a plan for more steps?
If it's just standard language, it should be "the following step" since there's just one.

Reply:
Done.